Dfe.PrepareConversions/Dfe.PrepareConversions/Pages/Shared/_OpenApplicationFormNewTab.cshtml

```diff
@@ -1,7 +1,7 @@
 ﻿@using Dfe.PrepareConversions.Data.Models
 @if (ViewData["AcademyTypeAndRoute"] is not null)
 {
-   @if (ViewData["AcademyTypeAndRoute"].Equals(AcademyTypeAndRoutes.Voluntary) || ViewData["AcademyTypeAndRoute"].Equals(AcademyTypeAndRoutes.FormAMat))
```

Comment:
Do we still want to show this for FAM's with an application or just handle it on the child as you have here?